convert/lib/convert.py

```diff
@@ -0,0 +1,614 @@
+                }
+            )
+            for s_type in ["Public", "Private", "Pod"]:
+                vpc_prefix = "" if s_type == "Pod" else "VPC"
+                prefix = f"{vpc_prefix}{{{{stack_name}}}}{s_type}Subnet{count+1}"
+                template["vpc_stack"].extend(
+                    [
+                        {
+                            "cf": f"{prefix}Subnet",
+                            "tf": f"aws_subnet.{s_type.lower()}[{count}]",
+                        },
+                        {
+                            "cf": f"{prefix}RouteTable",
+                            "tf": f"aws_route_table.{s_type.lower()}[{count}]",
+                        },
+                        {
+                            "cf_rtassoc": {
+                                "subnet": f"{prefix}Subnet",
+                                "route_table": f"{prefix}RouteTable",
+                            },
+                            "tf": f"aws_route_table_association.{s_type.lower()}[{count}]",
+                        },
+                    ]
+                )
+            template["vpc_stack"].extend(
+                [
+                    {
+                        "cf": f"VPC{{{{stack_name}}}}PublicSubnet{count+1}EIP",
+                        "tf": f"aws_eip.nat_gateway[{count}]",
+                    },
+                    {
+                        "cf": f"VPC{{{{stack_name}}}}PublicSubnet{count+1}NATGateway",
+                        "tf": f"aws_nat_gateway.public[{count}]",
+                    },
+                ]
+            )
+
+        if efs_backups:
+            template["efs_stack"].extend(efs_backup_resources)
+
+        if route53:
+            template["eks_stack"].append(route53_resource)
+
+        return template
+
+    def resource_map(self):
+        resource_map = self.generate_resource_map(
+            self.args.availability_zones, self.args.efs_backups, self.args.route53
+        )
+        print(yaml.safe_dump(resource_map))
+
+    def get_imports(self):
+        self.setup()
+        cdkconfig = yaml.safe_load(
+            [
+                o["OutputValue"]
+                for o in self.cf.describe_stacks(StackName=self.stack_name)["Stacks"][
+                    0
+                ]["Outputs"]
+                if o["OutputKey"] == "cdkconfig"
+            ][0]
+        )
+
+        if self.args.resource_map:
+            with open(self.args.resource_map) as f:
+                resource_map = yaml.safe_load(f)
+        else:
+            resource_map = self.generate_resource_map(
+                availability_zones=self.args.availability_zones
+                or cdkconfig["vpc"]["max_azs"],
+                efs_backups=cdkconfig["efs"]["backup"]["enable"],
+                route53=bool(cdkconfig["route53"]["zone_ids"]),
+            )
+
+        def t(val: str) -> str:
+            val = re.sub(r"{{region}}", self.region, val)
```

Comment:
You can actually use the built-in format function for this exactly if you only use one bracket:
```
>>> "{stack}".format(stack="hello")
'hello'
```

Reply:
This made it occur to me that, since we have a couple actual formatted input strings that this is in causing us to have to do `{{{{stack_name}}}}`, that probably the best thing to do is keep the current substitution method and change the sigil away from braces altogether. Four braces is madness, and two into a format gives me vietnam flashbacks of our deployer jinja turduckens.

I made it a `%`. Since we use `re` elsewhere, I'm leaving it as `re.sub` (as opposed to maybe doing `"".replace`, say).